src/webgpu/web_platform/util.ts

```diff
@@ -13,113 +13,284 @@ declare global {
   }
 }
 
-export const kVideoInfo =
-  /* prettier-ignore */ makeTable(
-                                             ['mimeType'                     ] as const,
-                                             [undefined                      ] as const, {
-    // All video names
-    'four-colors-vp8-bt601.webm':            ['video/webm; codecs=vp8'       ],
-    'four-colors-theora-bt601.ogv':          ['video/ogg; codecs=theora'     ],
-    'four-colors-h264-bt601.mp4':            ['video/mp4; codecs=avc1.4d400c'],
-    'four-colors-vp9-bt601.webm':            ['video/webm; codecs=vp9'       ],
-    'four-colors-vp9-bt709.webm':            ['video/webm; codecs=vp9'       ],
-    'four-colors-vp9-bt2020.webm':           ['video/webm; codecs=vp9'       ],
```

Comment:
Oh, just noticed this file has disappeared from the list.

Reply:
Yes. And no correspond expectations in previous table. I removed it. Maybe we need a TODO for it.

Reply:
Ah OK. If we were not testing the bt2020 file at all then we should have a TODO for it. There might be one already. But we should also file an issue about it because that's where we go to look for work that needs to be done.

Reply:
I didn't realize we didn't have a bt2020 file at all. Glad we caught that this was missing :)